developers-italia.oas.yaml

```
@@ -695,7 +695,11 @@ components:
         id: o9ohv7d53cd5o
       type: object
       properties:
-        id:
+        organizationId:
```

I would use just `id`, "organization" might be confusing

```suggestion
        id:
```